content/concepts/tokens/nftoken-batch-minting.md

```diff
@@ -0,0 +1,42 @@
+---
+html: nftoken-batch-minting.html
+parent: non-fungible-tokens.html
+blurb: Minting NFTokens in batches.
+filters:
+ - include_code
+labels:
+ - Non-fungible Tokens, NFTs
+status: not_enabled
+---
+
+# Batch minting
+
+There are two common approaches to minting NFTokens in batches: mint on demand and scripted minting.
+
+## Mint On Demand (Lazy Minting)
+
+When using a mint on demand model, you and potential customers make buy or sell offers for initial sales of a NFToken off the XRP Ledger. When you are ready to commence the initial sale, you mint the token, create a sell offer or accept a buy offer, then complete the transaction.
+
+### Benefits
+
+* There is little or no reserve requirement for holding unsold NFTokens.
```

Comment:
I think you can make this statement stronger:

```suggestion
* There is no reserve requirement for holding unsold NFTokens.
```

After all, if they haven't been minted, those tokens don't exist on the ledger at all so they couldn't possibly contribute to your reserve requirement.